src/modules/quoiFeur/quoiFeur.helpers.ts

```diff
@@ -1,46 +1,25 @@
-import {
-  ChannelType,
-  type ChatInputCommandInteraction,
-  Client,
-  Guild,
-  type Message,
-  Role,
-} from 'discord.js';
+import { ChannelType, type ChatInputCommandInteraction, type Message } from 'discord.js';
 
 import { cache } from '../../core/cache';
 import { removeEmoji, removePunctuation } from '../../helpers/regex.helper';
 
 const ONE_MINUTE = 1 * 60 * 1000;
-const MUTED_ON_COUBEH = 'Muted on Coubeh';
 
 const quoiDetectorRegex = /\bquoi\s*$/i;
 const endWithQuoi = (text: string) => quoiDetectorRegex.test(removeEmoji(removePunctuation(text)));
 
-const reactWithFeur = async (message: Message) => {
-  await message.react('🇫');
-  await message.react('🇪');
-  await message.react('🇺');
-  await message.react('🇷');
+const reactWith = async (message: Message, reactions: string[]) => {
+  for (const reaction of reactions) {
+    await message.react(reaction);
+  }
 };
 
 const reactWithCoubeh = async (message: Message) => {
```

Comment:
I think it's best if we return the promise directly instead of awaiting inside here.